src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -35,15 +35,18 @@ export const RoomModal: React.FC<RoomModalProps> = ({ show, onHide, room }) => {
   const venue = useSelector(currentVenueSelector);
   const venues = useSelector(orderedVenuesSelector);
   const venueEvents = useSelector(venueEventsSelector) ?? [];
-  const users = usePartygoers();
+  const { recentWorldUsers } = useRecentWorldUsers();
 
   const venueName = venue?.name;
   const roomTitle = room?.title;
 
+  // @debt Separete room attendies into a separate hook
   const usersToDisplay = useMemo(
     () =>
-      users?.filter((user) => user.lastSeenIn?.[`${venueName}/${roomTitle}`]),
-    [users, venueName, roomTitle]
+      recentWorldUsers.filter(
+        (user) => user.lastSeenIn?.[`${venueName}/${roomTitle}`]
+      ),
+    [recentWorldUsers, venueName, roomTitle]
   );
```

Comment:
This would be resolved by implementing `useRecentVenueUsers` as described elsewhere in this review.